content/integrations/kubernetes.md

```diff
@@ -127,6 +127,24 @@ The Kubernetes check includes the following service checks:
 * `kubernetes.kubelet.check.syncloop`:
   If `CRITICAL` or `NO DATA`, Kubelet's sync loop that updates containers isn't working.
 
+* `kubernetes_state.node.ready`:
+  Returns `CRITICAL` if a cluster node is not ready. Returns `OK` otherwise.
+
+* `kubernetes_state.node.out_of_disk`:
+  Returns `CRITICAL` if a cluster node is out of disk space. Returns `OK` otherwise.
+
+* `kubernetes_state.node.disk_pressure`:
+  Returns `CRITICAL` if a cluster node is in a disk pressure state. Returns `OK` otherwise.
+
+* `kubernetes_state.node.memory_pressure`:
+  Returns `CRITICAL` if a cluster node is in a memory pressure state. Returns `OK` otherwise.
+
+* `kubernetes_state.node.network_unavailable`:
+  Returns `CRITICAL` if a cluster node is in a network unavailable state. Returns `OK` otherwise.
+
+* `kubernetes_state.pod.phase`:
```

Comment:
Can we not document this one, we're trying to phase it out. We'll start sending a deprecation warning about it in 6.5, and we'll turn it off by default in 6.6.